pkg/api/bytes.go

```diff
@@ -27,6 +27,9 @@ type bytesPostResponse struct {
 // bytesUploadHandler handles upload of raw binary data of arbitrary length.
 func (s *server) bytesUploadHandler(w http.ResponseWriter, r *http.Request) {
 	ta := s.createTag(w, r)
+	if ta == nil {
+		return
```

Comment:
I would skip this check or add an internal server error response.

Reply:
CreateTag already creates a internal response Error. Here we just dont proceed so that the error is sent back as response.